docs/api/ctx.md

```diff
@@ -1852,6 +1852,47 @@ app.Get("/", func(c fiber.Ctx) error {
 })
 ```
 
+## SendStreamWriter
```

Comment:
please mention this new feature with example in our what's new section
(please not as a bullet point -> I still have to formulate it - in the right proportions like the other examples)

Reply:
For sure, I have added an entry to /docs/whats_new.md for `SendStreamWriter`